ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-upgrades.yaml

```diff
@@ -0,0 +1,26 @@
+releases:
+  initial:
+    release:
+      channel: stable
+      version: "4.4"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.5"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-upgrade-informer
```

Comment:
Why isn't there an `OPENSHIFT_UPGRADE=true` env here?

Reply:
Now it's set.